addons/calendar/static/src/views/attendee_calendar/attendee_calendar_controller.xml

```diff
@@ -43,5 +43,8 @@
                 </t>
             </div>
         </xpath>
+        <xpath expr="//t[@t-elif]//span[hasclass('badge bg-100 rounded')]" position="attributes">
+            <attribute name="t-if" separator="and">!env.isSmall</attribute>
```

Comment:
Thanks for quick fix!!
suggestion 
```suggestion
            <attribute name="t-if" add="and !env.isSmall" separator=" "/>

Reply:
hello @kpt-odoo,
thanks for the suggestion i have changed attribute tag to like this 
```xml
<attribute name="t-if" add="!env.isSmall" separator="and"/>
```

Reply:
Okay